app/models/user.rb

```diff
@@ -101,9 +101,12 @@ def with_team_kind(kind)
     end
 
     def with_all_associations_joined
-      includes(:conferences).group("conferences.id").
-      includes(:roles).group("roles.id").
-      includes(roles: :team).group("teams.id")
+      eager_load(:conferences).
```

Comment:
I have to read up on that `eager_load` call. Are you sure it does the same thing? All the group clauses are missing.